src/elm/Help/Help.elm

```diff
@@ -50,13 +50,16 @@ view : Args msg -> Html msg
 view args =
     li
         [ id "contextual-help"
+        , attribute "aria-label" "toggle contextual help for this page"
```

Comment:
needed to promote this element to be tab focusable, because we are removing the tab focus from the button itself, due to the manual click handler that circumvents the built-in details click event